src/queryResult/queryResultWebViewController.ts

```diff
@@ -301,6 +302,14 @@ export class QueryResultWebviewController extends ReactWebviewViewController<
             .get(Constants.extConfigResultKeys.ResultsFontFamily) as string;
     }
 
+    public getDefaultViewModeConfig(): qr.QueryResultViewMode {
+        const configValue = this.vscodeWrapper
+            .getConfiguration(Constants.extensionName)
+            .get("defaultQueryResultsViewMode") as string;
+
+        return configValue === "text" ? qr.QueryResultViewMode.Text : qr.QueryResultViewMode.Grid;
```

Comment:
Maybe make this an enum so it's more explicitly kept in sync with package.json values